### ISSUE TITLE
Problem about training on a small trainset

### ISSUE THREAD

**Issue by JINXvvv** [facebookresearch/supervision-by-registration#67]
I have trained a model on 300W and it works properly and the training loss is about 60.But when I decrease the number of trainset(200 images),the training loss is about 130.I have never modify the code!
PS: I decrease the number of trainset for some other use.

**Comment by D-X-Y** [comment]
Our base detector is CPM. 200 images might be insufficient to train it. Why not combine 300W with your new training set to train? :)
